src/olympia/shelves/models.py

```diff
@@ -27,7 +31,9 @@ class Shelf(ModelBase):
     footer_text = models.CharField(
         max_length=70,
         blank=True,
-        help_text='e.g., See more recommended extensions. Will be translated.',
+        help_text='e.g., See more recommended extensions. Will be translated. '
+        '`random-tag` shelves can use {tag} in the text, which will be substituted for '
+        'the random tag selected.',
```

Comment:
Isn't django generating a migration file for this kind of changes?

Reply:
it would if the migration command was run, but as it's a no-op at the database level we don't generally do it.  I could though?

Reply:
We usually commit these migration files eventually so maybe we could add it here to avoid having to add it later yeah